draft-documents/a-roadmap-for-ssi.md

```diff
@@ -1,346 +1,166 @@
+
+### Preventing Technical Debit
+
+This perfect storm arrives at a time when non-SSI alternatives are also being considered at scale. If these alternatives gain traction inside key adoption communities like DMVs, which are historically slow to change, our failure to focus on SSI adoption will indirectly allow for new alternatives to get entrenched thereby making our adoption challenge more difficult. For example, one of our core worldwide identity instruments, the digital driver license, is being considered by national and state governments worldwide. Unfortunately, the new mobile driver license (mDL) initiative that is being considered by [ISO](https://www.iso.org/standard/41920.html) will yield technical debit within our society that will make the adoption of SSI even more difficult.  The time is now to get focused on an clear and achievable rollout roadmap for SSI technology.    
+ 
+### Resetting Priorities
+An honest assessment of the state of our SSI initiative will yield the reality that we are “not ready for prime time”. If we can get the community to agree on short and long term priorities, we have a very good chance to succeed and prevent unnecessary adoption hurdles. So instead of worrying about when/if DNA chips may be used to track humans, we should focus on:  
+
+* Setting technology available expectations so early adopters can develop plans 
+* Hiding the complexity for usage; 
+* Solving the Wallet distribution problem
+	* Who are the wallet makers 
+	* How wallets will be [interoperable and secure](http://www.links.org/files/nspw36.pdf) 
+
+### Debunking Myths
+
+The urgency for a roadmap may be justified best by listening to the naysayers. An unfortunate outcome of a poor communication plan is the rise of myths that need to be discredited. Here are a few examples of unfortunate myths that are already circulating in the media and amongst analysts:
+
+* There is a perception that blockchain based identity solutions actually store personally identifiable information (PII) on the distributed ledger. This is utterly false. The SSI initiative not only advocates against such poor practices, the community has established several principles including that of *portability* which addresses this issue. A globally unique [decentralized identifier](https://w3c-ccg.github.io/did-spec/#dfn-did) (DID), which is compliant with internet standard [Universally Unique Identifiers](https://en.wikipedia.org/wiki/Universally_unique_identifier) (UUIDs), is used to establish a non-correlatable unique ID. While this DID may be stored on a blockchain and used as mechanism for public key exchange, the actual PII associated with an individual is never placed on a public ledger. This DID is used by an SSI solution to deliver public keys that can be used to verify identity realted credentials that are exchanged privately between willing participants at the edges of the network.
```

Comment:
Start with "For example," and use a different word than "perception" -- "For example, there is a misconception that..."

Reply:
Made changed and submitted another pull request.